job-dsls/jobs/deploy_jobs.groovy

```diff
@@ -194,7 +194,10 @@ for (repoConfig in REPO_CONFIGS) {
             }
         }
 
-        if (repo == "optaplanner") {
+        if (repo == "optaplanner" ||
```

Comment:
I propose you to change it by
`if (["optaplanner", "optaweb-employee-rostering", "optaweb-vehicle-routing", "optaplanner-quickstarts"].contains(repo))`

Reply:
or we can define array variable and do the calling method on it as it would be more readable and clean IMHO

something like
```
def OPTA_REPOS = ["optaplanner", "optaweb-employee-rostering", "optaweb-vehicle-routing", "optaplanner-quickstarts"];
if (OPTA_REPOS.contains(repo)) ...
```